static/js/containers/PaymentPage.js

```diff
@@ -95,6 +109,118 @@ class PaymentPage extends React.Component {
     )
   );
 
+  handleOrderStatus = (): void => {
+    const { klasses } = this.props;
+    const query = new URI().query(true);
+    if (!klasses.loaded) {
+      // wait until we have access to the klasses
+      return;
+    }
+
+    const status = query.status;
+    if (status === 'receipt') {
+      const orderId = parseInt(query.order);
+      const klass = getKlassWithFulfilledOrder(klasses.data, orderId);
+      if (klass) {
+        this.handleOrderSuccess(klass);
+      } else {
+        this.handleOrderPending();
+      }
+    } else if (status === 'cancel') {
+      this.handleOrderCancellation();
+    }
+  };
+
+  handleOrderSuccess = (klass: Object): void => {
+    const { dispatch, ui: { toastMessage } } = this.props;
+    if (toastMessage === null) {
+      dispatch(setToastMessage({
+        title: 'Order Complete!',
+        message: `You are now enrolled in ${klass.full_klass_title}`,
```

Comment:
This message probably does not make much sense. We should say something like "you made a successful payment for Bootcamp FooBar" or something like this.
Maybe ask Rob